app/src/main/java/co/techmagic/hr/presentation/time_tracker/HrAppTimeTrackerPresenter.kt

```diff
@@ -50,7 +50,7 @@ class HrAppTimeTrackerPresenter(
 
         call(timeTrackerInteractor.isRunning(), {
             it.report?.date?.toCalendar()?.let { date ->
-                onDateSelected(date)
+                onDateSelected(date.dateOnly())
```

Comment:
I think it makes sense to extract date data only inside the `onDateSelected` method to prevent possible issues in the future. What do you think?